embeddings/embedding/auto_flair.py

```diff
@@ -56,3 +70,42 @@ def from_hub(repo_id: str, **kwargs: Any) -> FlairEmbedding:
         except EnvironmentError:
             AutoFlairDocumentEmbedding._log_info_static(repo_id)
             return AutoStaticDocumentEmbedding.from_default_config(repo_id, **kwargs)
+
+
+class AutoFlairDocumentPoolEmbedding(AutoFlairEmbedding):
```

Comment:
i am not a big an of an pooling embedding being  separate class that do the logic of pooling, in my opinion it should be more in a away that pooling is a transformation, or we have a class that accepts embedding source and abstract class implementation that is responsible for pooling, that way we can easily extend the pooling methods and apply them to other models

Reply:
I would also discuss it seperately, not this in scope of this PR, as we need to decide whether we use Flair Pooling methods or we implement our own scheme. It is important issue in case of static embedding methods. Nevertheless this may be a subject of change in further PR's.